rero_ils/static/scss/rero_ils/styles.scss

```diff
@@ -53,16 +54,25 @@ ng\:form {
 }
 
 .nopadding {
-   padding: 0 !important;
-   margin: 0 !important;
+  padding: 0 !important;
```

Comment:
can you replace this by bootstrap class?

Reply:
This class is not even used. Will remove it.